packages/frontend/src/components/SchedulersView/LogsView.tsx

```diff
@@ -0,0 +1,339 @@
+                                                    <Text color="white" span>
+                                                        {item.format === 'csv'
+                                                            ? 'CSV'
+                                                            : 'Image'}
+                                                    </Text>
+                                                </Text>
+                                                <Text fz={12} color="gray.5">
+                                                    Created by:{' '}
+                                                    <Text color="white" span>
+                                                        {user?.firstName}{' '}
+                                                        {user?.lastName}
+                                                    </Text>
+                                                </Text>
+                                            </Stack>
+                                        }
+                                    >
+                                        <Text
+                                            fw={600}
+                                            lineClamp={1}
+                                            sx={{
+                                                overflowWrap: 'anywhere',
+                                                '&:hover': {
+                                                    textDecoration: 'underline',
+                                                },
+                                            }}
+                                        >
+                                            {item.name}
+                                        </Text>
+                                    </Tooltip>
+                                    <Text fz={12} color="gray.6">
+                                        {chartOrDashboard?.name}
+                                    </Text>
+                                </Stack>
+                            </Group>
+                        </Anchor>
+                    );
+                },
+                meta: {
+                    style: {
+                        width: 250,
+                    },
+                },
+            },
+            {
+                id: 'jobs',
+                label: 'Job',
+                cell: (item) => {
+                    return currentLogs(item, logs).length > 0 ? (
+                        <Box>
+                            <Group spacing="two">
+                                <Text fz={12} fw={500}>
+                                    All jobs
+                                </Text>
+                                <ActionIcon onClick={toggle} size="sm">
+                                    <MantineIcon
+                                        icon={IconChevronDown}
+                                        color="black"
+                                        size={13}
```

Comment:
does `sm` look too bad? I feel like we are still using a lot of custom sizes and trying to understand the reason

Reply:
unfortunately `sm` is too small and `md` too big :/